src/views/WafRules/FormFields/FormFieldsWafRules.vue

```diff
@@ -0,0 +1,343 @@
+<script setup>
+  import { ref } from 'vue'
+  import FormHorizontal from '@/templates/create-form-block/form-horizontal'
+  import InputText from 'primevue/inputtext'
+  import InputSwitch from 'primevue/inputswitch'
+  import Card from 'primevue/card'
+  import Dropdown from 'primevue/dropdown'
+
+  import { useField } from 'vee-validate'
+  defineOptions({ name: 'form-fields-variables' })
+
+  const { value: name, errorMessage: nameError } = useField('name')
+  const { value: active } = useField('active')
+  const { value: crossSiteScriptingSensitivity } = useField('crossSiteScriptingSensitivity')
+  const { value: directoryTraversalSensitivity } = useField('directoryTraversalSensitivity')
+  const { value: evadingTricksSensitivity } = useField('evadingTricksSensitivity')
+  const { value: fileUploadSensitivity } = useField('fileUploadSensitivity')
+  const { value: identifiedAttackSensitivity } = useField('identifiedAttackSensitivity')
+  const { value: remoteFileInclusionSensitivity } = useField('remoteFileInclusionSensitivity')
+  const { value: sqlInjectionSensitivity } = useField('sqlInjectionSensitivity')
+  const { value: unwantedAccessSensitivity } = useField('unwantedAccessSensitivity')
+  const { value: fileUpload } = useField('fileUpload')
+  const { value: evadingTricks } = useField('evadingTricks')
+  const { value: unwantedAccess } = useField('unwantedAccess')
+  const { value: identifiedAttack } = useField('identifiedAttack')
+  const { value: crossSiteScripting } = useField('crossSiteScripting')
+  const { value: directoryTraversal } = useField('directoryTraversal')
+  const { value: remoteFileInclusion } = useField('remoteFileInclusion')
+  const { value: sqlInjection } = useField('sqlInjection')
+
+  const sensitivity = ref([
+    { name: 'Sensitivity Highest', value: 'highest' },
+    { name: 'Sensitivity High', value: 'high' },
+    { name: 'Sensitivity Medium', value: 'medium' },
+    { name: 'Sensitivity Low', value: 'low' },
+    { name: 'Sensitivity Lowest', value: 'lowest' }
+  ])
+</script>
+
+<template>
+  <FormHorizontal
+    title="General"
+    description=""
+  >
+    <template #inputs>
+      <div class="flex flex-col sm:max-w-lg w-full gap-2">
+        <label
+          for="name"
+          class="text-color text-base font-medium"
+          >Name *
+        </label>
+        <InputText
+          
+          v-model="name"
+          type="text"
+          id="name"
+          :class="{ 'p-invalid': nameError }"
+        />
+        <small
+          v-if="nameError"
+          class="p-error text-xs font-normal leading-tight"
+        >
+          {{ nameError }}
+        </small>
```

Comment:
Você pode utilizar o component FieldText, ficando algo assim:
```suggestion
        <FieldText
          label="Name *"
          name="name"
          :value="name"
        />
```